README.md

```diff
@@ -984,6 +984,7 @@ _Packages that help with building Distributed Systems._
 - [glow](https://github.com/chrislusf/glow) - Easy-to-Use scalable distributed big data processing, Map-Reduce, DAG execution, all in pure Go.
 - [gmsec](https://github.com/gmsec/micro) - A Go distributed systems development framework.
 - [go-doudou](https://github.com/unionj-cloud/go-doudou) - A gossip protocol and OpenAPI 3.0 spec based decentralized microservice framework. Built-in go-doudou cli focusing on low-code and rapid dev can power up your productivity.
+- [go-eagle](https://github.com/go-eagle/eagle) - A Go framework for the API or Microservice with handy scaffolding tools.
```

Comment:
Hey! 👋🏻 Just checked out the framework. Looks supercool. Something that I should definitely check out. 

Could you please also add the following information in your PR Description, for quicker review from the maintainers? I happened to found them missing the URLs that are required.

![image](https://github.com/user-attachments/assets/72cf91b4-dfc3-4de1-b957-3cb4de4592f4)

Reply:
Thanks @ashwingopalsamy @qloog ❤️